porcupine/tabs.py

```diff
@@ -401,7 +407,7 @@ def _import_lexer_class(name: str) -> LexerMeta:
 
 @dataclasses.dataclass
 class ReloadInfo(utils.EventDataclass):
-    was_modified: bool
+    was_modified: bool  # FIXME: rename to had_unsaved_changes
```

Comment:
Make separate pr and delete fixme

```suggestion
    was_modified: bool
```